### PR TITLE
Edit certs -> certificates in user-facing text.

### PR DESCRIPTION
**Pull request by jsha**
To reduce confusion, we should consistently use the full term.

### CODE REVIEW

**Comment by alexzorin** [comment]
I agree. I've found a few other notable ones:

https://github.com/certbot/certbot/blob/fcc8b38c02cc10ff320b401ffa242f2253ce9552/certbot/certbot/_internal/cert_manager.py#L372

https://github.com/certbot/certbot/blob/fcc8b38c02cc10ff320b401ffa242f2253ce9552/certbot/certbot/_internal/main.py#L256-L257

https://github.com/certbot/certbot/blob/fcc8b38c02cc10ff320b401ffa242f2253ce9552/certbot/certbot/_internal/main.py#L437-L438

https://github.com/certbot/certbot/blob/fcc8b38c02cc10ff320b401ffa242f2253ce9552/certbot/certbot/_internal/main.py#L516-L517

(though that one is going to get rewritten by #8513 anyway)

https://github.com/certbot/certbot/blob/fcc8b38c02cc10ff320b401ffa242f2253ce9552/certbot/certbot/_internal/main.py#L600-L601

https://github.com/certbot/certbot/blob/fcc8b38c02cc10ff320b401ffa242f2253ce9552/certbot/certbot/_internal/renewal.py#L484-L487

https://github.com/certbot/certbot/blob/fcc8b38c02cc10ff320b401ffa242f2253ce9552/certbot-dns-digitalocean/certbot_dns_digitalocean/_internal/dns_digitalocean.py#L22

https://github.com/certbot/certbot/blob/fcc8b38c02cc10ff320b401ffa242f2253ce9552/certbot-dns-linode/certbot_dns_linode/_internal/dns_linode.py#L27

There are further occurrences but those are in rarely hit code paths. You can see them with:

    semgrep --exclude "*_test.py" --exclude "test_*.py"  -l py -e '"=~/\\scerts?\\b/"' | grep -v '"""'

If you don't have time to chase these down I'd be happy to do so as part of other UX changes.

**Comment by jsha** [comment]
Thanks for tracking down those others! I'm surprised they didn't turn up in my search. If you have time to incorporate those additional fixes, I'd appreciate it!

File: certbot/certbot/_internal/storage.py [hunk 1]
```diff
@@ -809,8 +809,8 @@ def ensure_deployed(self):
         May need to recover from rare interrupted / crashed states."""
 
         if self.has_pending_deployment():
-            logger.warning("Found a new cert /archive/ that was not linked to in /live/; "
-                        "fixing...")
+            logger.warning("Found a new certiifcate /archive/ that was not "
```
Comment (jsha) [inline certbot/certbot/_internal/storage.py:812]:
```suggestion
            logger.warning("Found a new certificate /archive/ that was not "
```

**Review by jsha — COMMENTED** [review]
Looks good to me! Can't approve since I'm the author.

**Review by ohemorange — APPROVED** [review]
lgtm, thanks!

**Pull request merged**